docs/rewards/rewards-troubleshooting.mdx

```diff
@@ -63,21 +63,39 @@ If the battery is low, the connection between your station and the M5 gateway wi
 
 ## Sensor Problems
 
-Weather data packets failed to pass our quality checks. We evaluate all data packets sent by our weather stations using our QoD (Quality of Data) algorithm. You can read more about this algorithm [here](https://docs.weatherxm.com/project/quality-of-data).
-There are several reasons that you may experience this error, which includes:
+We have discovered that the weather data packets received, did not meet our quality standards. 
+To ensure the accuracy of the information sent by our weather stations, we evaluate all data packets using our [Quality of Data (QoD) algorithm](/project/quality-of-data). For more information about this algorithm, please refer to this [here](/project/quality-of-data). 
+
+Below are some of the possible errors and the appropriate actions you can take to resolve the issues:
+
+
+#### Out of Bounds Data 
+It's possible that a sensor sends a completely irrational value, which implies to a faulty sensor. It is recommended that you observe it for a few days before reaching out to our [support team](https://support.weatherxm.com)  for sensor replacement.
+
+#### Spikes on Data 
+You should first check your station if it is optimally deployed and then you should contact [support](https://support.weatherxm.com). Take a look at our [Deployment Examples](/deployment-examples) and [Best Practices](/wxm-devices/wifi-m5-bundle/install-weather-station#best-practices-for-proper-station-installation) to ensure that your station is deployed correctly.
+
 
-- Out of bounds data
-- Spikes on data 
-- Constant values
-- Frozen sensor
 
-Check out our [Deployment Examples](/deployment-examples) and [Best Practices](/wxm-devices/wifi-m5-bundle/install-weather-station#best-practices-for-proper-station-installation) to ensure that your station is deployed correctly,
-and observe it for a couple of days before contacting our support team. If you still receive the error, you are advised to contact our [support team](https://support.weatherxm.com) for more guidance.
+#### Constant Values 
+You should first check your station if it is optimally deployed and then you should contact [support](https://support.weatherxm.com). Take a look at our [Deployment Examples](/deployment-examples) and [Best Practices](/wxm-devices/wifi-m5-bundle/install-weather-station#best-practices-for-proper-station-installation) to ensure that your station is deployed correctly.
```

Comment:
Please explain what constant values means (one or more of your sensors report constant values which implies a bad installation, or less often, a problematic sensor).
If your station is optimally installed and the constant value errors persist for a few days, do the above power cycle and if it doesn't go away, contact suhpport.